site/src/components/textarea.md

```diff
@@ -41,7 +41,7 @@ Here is an example of a standard textarea usage paired with a label.
 
 `textarea-danger`/`textarea-success` helps with visual reflection of validation states.
 
-| State   | Class                         |                                                                                   |
+| State   | Class                         | Example                                                                           |
```

Comment:
Should "Example" be "Code" instead to match other pages?

Reply:
Nice catch! I changed them all to Example, since they are not referring to Code.